packages/cli/src/databases/migrations/sqlite/index.ts

```diff
@@ -17,6 +17,7 @@ import { IntroducePinData1654089251344 } from './1654089251344-IntroducePinData'
 import { AddNodeIds1658930531669 } from './1658930531669-AddNodeIds';
 import { AddJsonKeyPinData1659888469333 } from './1659888469333-AddJsonKeyPinData';
 import { CreateCredentialsUserRole1660062385367 } from './1660062385367-CreateCredentialsUserRole';
+import { AddWorkflowVersion1664567980717 } from './1664567980717-AddWorkflowVersion';
```

Comment:
Missing Postgres and MySQL migrations

Reply:
Workflow-Version got removed so the comment is outdated.